helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
-appVersion: 1.17.2
+appVersion: 1.22.1
 description: Deploy components for the aws-node-termination-handler app
-name: aws-node-termination-handler-app
+name: aws-node-termination-handler
 version: "[[ .Version ]]"
```

Comment:
It'd be cool to start using `abs`, but I understand it's out of the scope of this task.

Reply:
nah, I will try to do that, this app is not used anywhere so I can try it